CHANGELOG.md

```diff
@@ -4,6 +4,107 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.13.0 - 2019-02-19
+
+### Added
+
+- When cluster-creation times out waiting for cluster-version
+  completion, the installer now logs the last failing-operator
+  message (if any).
+- The installer now invokes the [cluster-config
+  operator][cluster-config-operator] on the bootstrap machine to
+  generate `config.openshift.io` resources.
+
+### Changed
+
+- The install-config version has been bumped from `v1beta2` to
+  `v1beta3` as `machines` has been split into `controlPlane` and
+  `compute`.  All users will need to update any saved
+  `install-config.yaml` to use the new schema.  As part of this
+  change, multiple compute pools are now supported (previously, only a
+  single `worker` pool was supported).
+
+    On libvirt, the `masterIPs` property has been removed, since you
+    cannot configure master IPs via the libvirt machine API provider.
+
+    On OpenStack, there is also a new `lbFloatingIP` property, which
+    allows you to provide an IP address to be used by the load
+    balancer.  This allows you to create local DNS entries ahead of
+    time before calling `create cluster`.
+
+- Cluster domain names have been adjusted so that the cluster lives
+  entirely within a per-cluster subdomain.  This keeps split-horizon
+  DNS from masking other clusters with the same base domain.
+- The cluster-version update URL has been changed from the dummy
+  `http://localhost:8080/graph` to the functioning
+  `https://api.openshift.com/api/upgrades_info/v1/graph` and the
+  channel has been changed from `fast` to `stable-4.0`, to opt
+  clusters in to 4.0 upgrades.
+- Machine-API resources have been moved from `cluster.k8s.io` to
+  `machine.openshift.io` to clarify our divergence from the upstream
+  types while they are unstable.  The `openshift-cluster-api`
+  namespace has been replaced with `openshift-machine-api` as well.
+- The installer now uses etcd and OS images referenced by the update
+  payload when configuring the machine-config operator.
+- The Kubernetes certificate authority is now self-signed, decoupling
+  its chain of trust from the root certificate authority.
+- The installer no longer creates a bootstrap service certificate
```

Comment:
`bootstrap service certificate` is not correct representation of the dropped `service ca`.
- it was not bootstrap only
- it is used to create certs for services. ie `service serving CA`

Reply:
> * it was not bootstrap only

I was going for "the one we create to bootstrap the cluster, because the cluster didn't create its own".  But sure, I can reword.

Reply:
> `bootstrap service certificate` is not correct...

Reworded with b6882f2 -> 8f08508.